feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.ts

```diff
@@ -48,10 +48,16 @@ export class ConfiguratorProductTitleComponent {
     this.showMore = !this.showMore;
   }
 
+  /**
+   * @deprecated since 3.1
+   */
```

Comment:
can you add a todo here or change it to 3.3 (currently we plan to release with 3.3)

Reply:
Done!